src/syntax/instrumentPpx.ml

```diff
@@ -87,14 +116,157 @@ let wrap_expr k e =
     else
       let marked = List.mem line c.CommentsPpx.marked_lines in
       let marker_file = !Location.input_name in
-      match marker marker_file ofs k marked with
+      match marker must_be_unique marker_file ofs k marked with
       | Some w -> Exp.sequence ~loc w e
       | None   -> e
 
+(* Given a pattern and a location, transforms the pattern into a list of
+   patterns by transforming or-patterns into cases (i.e., promoting them to the
```

Comment:
What's wrong with it? It's just indicative instead of imperative, like some of the older comments in the file.